test/klee-stats/output-topinsts.c

```diff
@@ -0,0 +1,33 @@
+// RUN: %llvmgcc %s -emit-llvm -O0 -c -o %t1.bc
+// RUN: rm -rf %t.klee-out0
+// RUN: rm -rf %t.klee-out1
+// RUN: rm -rf %t.klee-out2
+// RUN: %klee --output-dir=%t.klee-out0 --libc=klee --exit-on-error %t1.bc
+// RUN: %klee --output-dir=%t.klee-out1 --libc=klee --exit-on-error %t1.bc
+// RUN: %klee --output-dir=%t.klee-out2 --libc=klee --exit-on-error %t1.bc
+<<<<<<< HEAD
```

Comment:
BTW, you seem to be commenting on some inexistent code (shown as outdated on GitHub), which doesn't exist (and I did a single push).  Not sure what's going on here.